.github/workflows/python-release.yml

```diff
@@ -39,6 +39,4 @@ jobs:
         with:
           user: __token__
           password: ${{ secrets.PYPI_API_TOKEN }}
-          # TODO: remove repository_url to default to prod
-          repository_url: https://test.pypi.org/legacy/
```

Comment:
I manually [created a tag/release](https://github.com/looker-open-source/sdk-codegen/releases/tag/looker_sdk-v21.0.0) the same way release-please will in the future and then published it and saw the package go up on [testpypi](https://test.pypi.org/project/looker-sdk/21.0.0/)

I did a run in the examples/python/ dir using that test package to verify its integrity and all was good. So I'm blessing this now

I'll document it in a future 'sdk-codegen CI/CD runbook' but for now the trigger to send a version to pypi.org will be "publishing" the draft github release that release-please creates